scripts/openstack.py

```diff
@@ -0,0 +1,156 @@
+import argparse
```

Comment:
We had started to prefer docopt over argparse.  Is there a reason to not use docopt here?